test/unit/services/TriggerService.test.js

```diff
@@ -103,4 +103,48 @@ describe('The TriggerService', function () {
             }
         });
     });
+
+
+    describe('should validate fields using lodash template', function () {
```

Comment:
Do these tests actually run?
I don't see either `triggerCondition` or `forceRun` to satisfy `this.metTriggerCondition(oid, record, options) === "true"`?